modules/cli/src/main/scala/scala/cli/ScalaCli.scala

```diff
@@ -138,6 +139,7 @@ object ScalaCli extends CommandsEntryPoint {
   }
 
   private def main0(args: Array[String]): Unit = {
+    LauncherCLI.main(args)
```

Comment:
I do not like situation when calling a random method can side-effect in rest of the code not being run (`sys.exit`) and here this is a base of the design. 

I think we need at least a comment but I would love if this will be more obvious from the code.